src/components/staff-payment/StaffPayment.vue

```diff
@@ -168,15 +162,24 @@ export default class StaffPayment extends Vue {
   })
   readonly staffPaymentData!: StaffPaymentIF
 
-  /** Radio group model property. */
-  private paymentOption = StaffPaymentOptions.NONE
-
   /** FAS form model property. */
   private fasFormValid = false
 
+  /** If FAS form inputs are disabled. */
+  private get fasFormInputDisabled (): boolean {
```

Comment:
The `private` specifier may cause warnings in the template. Also, it doesn't matter if code outside this class calls a getter. So I recommend leaving computed variables as public.